src/main/java/bio/terra/common/iam/AuthenticatedUserRequest.java

```diff
@@ -0,0 +1,110 @@
+package bio.terra.common.iam;
+
+import bio.terra.common.exception.ApiException;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.util.Objects;
+import java.util.UUID;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+@JsonDeserialize(builder = AuthenticatedUserRequest.Builder.class)
+public class AuthenticatedUserRequest {
+
+  private final String email;
+  private final UUID reqId;
```

Comment:
if we're breaking backwards compatibility already, avoid abbreviations, use `requestId`. (if we can't remove this field entirely.) [ref](go/java-practices/naming#general-advice)

Reply:
The fact of moving the code to a different package will break on-disk compatibility in data serialized in Stairway.